shotover-proxy/tests/cassandra_int_tests/mod.rs

```
@@ -342,3 +349,46 @@ async fn test_cassandra_request_throttling() {
 
     batch_statements::test(&connection);
 }
+
+#[tokio::test(flavor = "multi_thread")]
+#[serial]
+async fn test_events_keyspace() {
+    let _docker_compose =
+        DockerCompose::new("example-configs/cassandra-passthrough/docker-compose.yml");
+
+    let _shotover_manager =
+        ShotoverManager::from_topology_file("example-configs/cassandra-passthrough/topology.yaml");
+
+    let user = "cassandra";
+    let password = "cassandra";
+    let auth = StaticPasswordAuthenticatorProvider::new(&user, &password);
+    let config = NodeTcpConfigBuilder::new()
+        .with_contact_point("127.0.0.1:9042".into())
+        .with_authenticator_provider(Arc::new(auth))
+        .build()
+        .await
+        .unwrap();
+
+    let session = TcpSessionBuilder::new(RoundRobinLoadBalancingStrategy::new(), config)
+        .build()
+        .unwrap();
+
+    let mut event_recv = session.create_event_receiver();
+
+    sleep(Duration::from_secs(3)).await; // let the driver finish connecting to the cluster and registering for the events
+
+    let create_ks = "CREATE KEYSPACE IF NOT EXISTS test_events_ks WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };";
+
+    session.query(create_ks).await.unwrap();
+
+    match timeout(Duration::from_secs(10), event_recv.recv()).await {
+        Ok(recvd) => {
+            if let Ok(event) = recvd {
```

I think this should be unwrapped to avoid the test silently failing.